README.md

```diff
@@ -0,0 +1,16 @@
+# Accelerator
+Accelerator replaces the default SRCDS crash handler with one that is a lot more reliable and uploads the crash reports to a community-accessible processing backend.
+This is useful since Valve's processing system discards crash reports from modded servers - and they still receive a representative sample of crash reports for actual game issues.
+
+The processing backend ([Throttle](https://crash.limetech.org/)) analyses crash reports to extract useful information, and in the case of common issues affecting many servers where manual analysis has been done, tags them with helpful notices with information on resolving the cause of the crash.
+
+## Installation
+- Click the Download link below.
```

Comment:
Why not make this the download link? Or use one of those fancy reference links for this area and also have it at the bottom of this section.